1-mandatory/1-writers.js

```diff
@@ -62,18 +64,41 @@ Exercise 2:
 */
 
```

Comment:
This is a nice solution 👍  A couple of small points here:
1. When you have code that looks like: `writer.alive === false` you can re-write it to `!writer.alive` - can you think of why that is?
2. In arrow functions, if the body of the function only has one line - you don't need the `{` `}` brackets and the `return`.
3. For practice, can you try writing this using method chaining? This means you won't need the `writersInTheirFortiesIsNotAlive` variable.